CHANGELOG.md

```diff
@@ -125,6 +125,7 @@ These functions and structs can be referenced in both code and blueprints it may
 - Fixed a assertion being triggered on async loaded entities due to queuing some component addition.
 - Fixed a bug where consecutive invocations of CookAndGenerateSchemaCommandlet for different levels could fail when running the schema compiler.
 - Fixed an issue where GameMode values won't be replicated between server workers if it's outside their Interest
+- Fixed the exception that was thrown when adding and removing components in Spatial component callbacks.
```

Comment:
This should go in the .x.y.z section now as it won't land in 0.12

Reply:
Good to know. Fixed the changelog.